platforms/android/gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
Remove the .bat file.

Reply:
Thinking about this - it may stay in case someone sets up Android NDK on Windows.